R/pkg/NAMESPACE

```diff
@@ -6,10 +6,15 @@ importFrom(methods, setGeneric, setMethod, setOldClass)
 #useDynLib(SparkR, stringHashCode)
 
 # S3 methods exported
+export("sparkR.session.getOrCreate")
```

Comment:
Yeah I dont think we have any methods which accept a spark context in fact. So hiding it / not having an API where we can pass a SparkContext around is fine by me.